src/gluonts/model/rotbaum/_estimator.py

```diff
@@ -0,0 +1,52 @@
+# Copyright 2018 Amazon.com, Inc. or its affiliates. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License").
+# You may not use this file except in compliance with the License.
+# A copy of the License is located at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# or in the "license" file accompanying this file. This file is distributed
+# on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either
+# express or implied. See the License for the specific language governing
+# permissions and limitations under the License.
+
+# First-party imports
+from gluonts.core.component import validated
+from gluonts.dataset.common import Dataset
+from gluonts.model.estimator import Estimator, Predictor
+
+# Relative imports
+from ._predictor import TreePredictor
+
+
+class ThirdPartyEstimator(Estimator):
```

Comment:
We might think about putting this to `gluonts.model` directly.

Reply:
Can you expatiate? I'm not sure I understand what you mean.